apps/api_web/lib/api_web/views/schedule_view.ex

```diff
@@ -61,10 +61,24 @@ defmodule ApiWeb.ScheduleView do
     State.Prediction.prediction_for(schedule, date)
   end
 
+  def arrival_time(
+        %{drop_off_type: 1, departure_time: seconds_past_midnight},
+        %{assigns: %{api_version: ver}} = conn
+      )
+      when ver < "2019-07-01",
+      do: format_time(seconds_past_midnight, conn)
+
   def arrival_time(%{arrival_time: seconds_past_midnight}, conn) do
     format_time(seconds_past_midnight, conn)
   end
 
+  def departure_time(
+        %{pickup_type: 1, arrival_time: seconds_past_midnight},
```

Comment:
I think this is right, but it's more forward-compatible if the match is `departure_time: nil, arrival_time: seconds` (and vice versa). That way, if there's another reason we have `nil` times in the future, older versions will still be supported.

Reply:
Well, if there's another reason we have `nil`, we'd probably want to have different logic to handle it. Anyway, changed like you say.